src/test/java/redis/clients/jedis/tests/commands/ListCommandsTest.java

```diff
@@ -129,9 +128,7 @@ public void lrange() {
     range = jedis.lrange("foo", 1, 2);
     assertEquals(expected, range);
 
-    expected = new ArrayList<String>();
-    range = jedis.lrange("foo", 2, 1);
-    assertEquals(expected, range);
+    assertNull(jedis.lrange("foo", 2, 1));
```

Comment:
is that a backward issue in Redis 6?

Reply:
Yes, a backward issue in Redis 6.
PS: Not a backward issue for Jedis.

Reply:
I think it should be fixed before 6.x will be released